students/lauraannf/lessons/lesson01/assignment/test_unit.py

```diff
@@ -0,0 +1,146 @@
+        other_input = ['300', 'description', 'rentalprice', 'n', 'n']
+        with patch('builtins.input', side_effect=furniture_input):
+            test_inventory = add_new_item()
+            test_return_furniture = {'100': {'product_code': '100',
+                                             'description': 'description',
+                                             'market_price': 24,
+                                             'rental_price': 'rentalprice',
+                                             'material': 'material',
+                                             'size': 'size'}}
+            self.assertEqual(test_return_furniture, test_inventory)
+        with patch('builtins.input', side_effect=electric_input):
+            test_inventory = add_new_item()
+            test_return_electrics = {'100': {'product_code': '100',
+                                               'description': 'description',
+                                               'market_price': 24,
+                                               'rental_price': 'rentalprice',
+                                               'material': 'material',
+                                               'size': 'size'},
+                                       '200': {'product_code': '200',
+                                               'description': 'description',
+                                               'market_price': 24,
+                                               'rental_price': 'rentalprice',
+                                               'brand': 'brand',
+                                               'voltage': 'voltage'}}
+            self.assertEqual(test_return_electrics, test_inventory)
+        with patch('builtins.input', side_effect=other_input):
+            test_inventory = add_new_item()
+            test_return_other = {'100': {'product_code': '100',
+                                         'description': 'description',
+                                         'market_price': 24,
+                                         'rental_price': 'rentalprice',
+                                         'material': 'material',
+                                         'size': 'size'},
+                                 '200': {'product_code': '200',
+                                         'description': 'description',
+                                         'market_price': 24,
+                                         'rental_price': 'rentalprice',
+                                         'brand': 'brand',
+                                         'voltage': 'voltage'},
+                                 '300': {'product_code': '300',
+                                         'description': 'description',
+                                         'market_price': 24,
+                                         'rental_price': 'rentalprice'}}
+            self.assertEqual(test_return_other, test_inventory)
+
+    def test_item_info(self):
```

Comment:
This test is currently shows an assertion error when you run it:

`$ python -m unittest test_unit.py -v`

See my comments below on why it's failing.